patterns/shop-by-price.php

```diff
@@ -47,14 +50,17 @@
 		</div>
 		<!-- /wp:cover -->
 
-		<!-- wp:paragraph {"align":"center","style":{"spacing":{"margin":{"top":"10px","right":"10px","bottom":"10px","left":"10px"}},"color":{"text":"#000000"}},"fontSize":"small"} -->
-		<p class="has-text-align-center has-text-color has-small-font-size" style="color:#000000;margin-top:10px;margin-right:10px;margin-bottom:10px;margin-left:10px"><strong>Summer Dinning</strong></p>
-		<!-- /wp:paragraph -->
+		<!-- wp:buttons {"layout":{"type":"flex","justifyContent":"center","verticalAlignment":"center"}} -->
+		<div class="wp-block-buttons">
+			<!-- wp:button {"textColor":"contrast","style":{"border":{"width":"0px","style":"none"},"spacing":{"padding":{"left":"0","right":"0","top":"0","bottom":"0"}}},"className":"is-style-outline","fontSize":"small"} -->
+			<div class="wp-block-button has-custom-font-size is-style-outline has-small-font-size"><a class="wp-block-button__link has-contrast-color has-text-color wp-element-button" href="<?php echo esc_url( wc_get_page_permalink( 'shop' ) ); ?>" style="border-style:none;border-width:0px;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><strong>Summer Dinning</strong></a></div>
```

Comment:
We should be able to pass along a query param in this link URL to display a filtered result that matches the price point. For example, this column is items "Under $15" so we can pass `?max_price=14` to the link.

Not a blocker, just noting this for a potential follow-up.